src/client/controller/RoomsController.js

```diff
@@ -45,6 +46,12 @@ RoomsController.prototype.attachEvents = function()
     this.repository.on('room:players', this.applyScope);
     this.repository.on('room:game', this.applyScope);
 
+    var self = this;
+    this.repository.on('rooms:nbPlayers', function(e) {
```

Comment:
Could not do the same as for `rooms` with `applyScope()`, as `nbPlayers` is a scalar type on the repository (so it was copied to $scope without reference).